### PR TITLE
Removed ServerCommon code from repo (replaced by package dependency)

### PR DESCRIPTION
**Pull request by xavierdecoster**
cc @skofman1 @scottbommarito @maartenba @ryuyu

### CODE REVIEW

**Comment by shishirx34** [comment]
Looks good!

File: src/NuGet.Services.Staging.Manager/project.json [hunk 1]
```diff
@@ -22,35 +22,35 @@
   },
 
   "dependencies": {
+    "EnterpriseLibrary.TransientFaultHandling.Data": "6.0.1304.1",
     "Microsoft.ApplicationInsights.AspNetCore": "1.0.0",
```
Comment (skofman1) [inline src/NuGet.Services.Staging.Manager/project.json:26]: > ApplicationInsights [](start = 15, length = 19)

can you please deploy this version after commit (for verification)?

**Comment by skofman1** [comment]
:shipit:

**Comment by scottbommarito** [comment]
LGTM

File: NuGet.Services.Staging.sln [hunk 2]
```diff
@@ -40,6 +38,8 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Client.Staging", "src
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Configuration", "src\NuGet.Services.Configuration\NuGet.Services.Configuration.xproj", "{E2D65F09-81A6-4D69-B128-C47C2DC43D36}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Logging", "src\NuGet.Services.Logging\NuGet.Services.Logging.xproj", "{3A70A370-5ABE-4A53-B5B9-297DB288964E}"
+EndProject
```
Comment (ryuyu) [inline NuGet.Services.Staging.sln]: Is there a reason we keep this reference in the sln file?

**Comment by ryuyu** [comment]
One question, otherwise LGTM.

File: NuGet.Services.Staging.sln (hunk 2 above)
Comment (xavierdecoster) [reply NuGet.Services.Staging.sln]: Nope, good catch. Classic "VS did not save my solution file" issue... :) Fixed in https://github.com/NuGet/NuGet.Staging/pull/72/commits/50333b13a8f3fd4579d72c0749d9a465eedbfd18

File: src/NuGet.Services.Staging.Manager/project.json (hunk 1 above)
Comment (xavierdecoster) [reply src/NuGet.Services.Staging.Manager/project.json:26]: Deployed to DEV environment. Please verify :)

**Pull request merged**